src/Components/Auth/Auth.jsx

```diff
@@ -0,0 +1,73 @@
+import { useState } from 'react';
+import './Auth.css';
+import { LoginComponent } from './Components/LoginComponent';
+import { SignUpComponent } from './Components/SignUpComponent';
+import { useSelector, useDispatch } from 'react-redux';
+import { setError } from '../../store/Auth/AuthSlice';
+
+const Auth = () => {
+	const dispatch = useDispatch();
+
+	const [showPassword, setShowPassword] = useState({ password: false, confirmPassword: false });
+	const { isLoginForm } = useSelector((store) => store.auth);
+	const defaultText = {
```

Comment:
Suggestion: You can store the defaultText into the separate file and then you can import and use it